cypress/page/shoping-cart.page.ts

```diff
@@ -0,0 +1,14 @@
+class ShopingCartPage {
+    private checkbtn: string;
+  
+    constructor() {
+      this.checkbtn = ".cart_navigation span";
+    }
+  
+    public pcheckout(): void {
+      cy.get(this.checkbtn).click();
+    }
+  }
+  
+  export {ShopingCartPage};
+  
```

Comment:
Be sure that this is the last line and there are no trailing spaces (That's why there is a warning of EOF)

Reply:
done @danielgalvis98